kratos/python_scripts/gid_output_process.py

```diff
@@ -645,3 +653,48 @@ def __remove_list_files(self):
                         os.remove(f)
                     except WindowsError:
                         pass
+                    
+    #
+    def __remove_post_results_files(self, step_label):
```

Comment:
The double underline __ is not something new in this file. If it is not desired would be better to correct it in all methods after this merge. Because this is mainly a file of @jcotela, maybe he can change it carefully later on.